warehouse/ingest-scripts/src/main/resources/bin/system/start-ingest.sh

```diff
@@ -29,7 +31,12 @@ else
   trap 'rm -f "$ingestHost"; exit $?' INT TERM EXIT
   echo $INGEST_HOST > $ingestHost
 
-  pdsh -f 25 -w ^${ingestHost} "$INGEST_BIN/ingest/start-ingesters.sh $FORCE" < /dev/null
+  localhost=$(hostname -s)
+  rm -rf /tmp/pdsh_log/${script_name}/$$
```

Comment:
/tmp is often on a small partition. Also, it gets cleaned up on reboot and at other times. Should we put these logs somewhere else? Maybe at least have a variable that defines the location so it can easily be customized for all scripts.

Reply:
I just realized I hadn't pushed my latest commit. Sorry about that! In the latest commit, I made the log location configurable with the default being to `${LOG_DIR}/pdsh_logs` defined in `dev.properties`.

Will push my commit soon.